src/php/inc/theme-widgets.php

```diff
@@ -6,14 +6,26 @@
  */
 
 /**
- * Register widget area.
+ * Register widgetized areas.
  */
 function sparkpress_theme_widgets_init() {
+	register_sidebar(
+		array(
+			'name'          => 'Primary Widget Area',
+			'id'            => 'primary-widget-area',
```

Comment:
I'd call this "Primary Sidebar" or something like that, especially since this one is pretty explicitly a sidebar.